googleCalender.js

```diff
@@ -0,0 +1,447 @@
+const max = 50;
+async function UsersFullSync({ dataStore, client }) {
+  console.log(" sync started");
+  const responsedata = await client.fetch("admin/directory/v1/users?customer=my_customer&projection=full&viewType=admin_view");
+  //console.log(JSON.stringify(responsedata));
+  if (!responsedata.ok) {
+    throw new Error(
+      `Accounts sync failed ${responsedata.status}:${responsedata.statusText}.`
+    );
+  }
+  let data = await responsedata.json();
+  // console.log(JSON.stringify(data));
+
+  for (let user of data.users) {
+    // console.log(JSON.stringify(user));
+    let user_data = {
+      agreed_to_terms: user.agreedToTerms,
+      archived: user.archived,
+      change_password_at_next_login: user.changePasswordAtNextLogin,
+      cocustomer_id: user.customerId,
+      etag: user.etag,
+      id: user.id,
+      include_in_global_address_list: user.includeInGlobalAddressList,
+      is_admin: user.isAdmin,
+      is_enforced_in_2_sv: user.isEnforcedIn2Sv,
+      is_mailbox_setup: user.isMailboxSetup,
+      kind: user.kind,
+      name_family_name: user.name.familyName,
+      name_full_name: user.name.fullName,
+      nname_given_name: user.name.givenName,
+      org_unit_path: user.orgUnitPath,
+      primary_email: user.primaryEmail,
+      recovery_email: user.recoveryEmail,
+      suspended: user.suspended,
+    };
+    // console.log(JSON.stringify(user_data))
+    //console.log(`user.primaryEmail${JSON.stringify(user.primaryEmail)}`);
+    dataStore.save("users", user_data);
+
+    await Events(dataStore, client, user.primaryEmail);
+
+  }
+}
+//const primaryEmail = ["user.primaryEmail"];
+async function Events(dataStore, client, primaryEmail) {
+ // console.log("primaryEmail");
+  const responsedata2 = await client.fetch(`calendar/v3/calendars/${primaryEmail}/events?maxResults=${max}&showDeleted=false&singleEvents=true&orderBy=startTime`);
+  console.log(JSON.stringify(responsedata2));
+
+  // console.log(error);
+  if (!responsedata2.ok) {
+    throw new Error(
+      `Accounts sync failed ${responsedata2.status}:${responsedata2.statusText}.`
+    );
+  }
+  //console.log(error);
+  let data2 = await responsedata2.json();
+  console.log(JSON.stringify(data2));
+  for (let ev of data2.items) {
```

Comment:
Provide meaning full name for variables. Follow the same for entire file.
```suggestion
  for (let items of data2.items) {
```